site/app/models/RainbowCustomizationJSON.php

```diff
@@ -70,7 +82,7 @@ public function __construct(Core $main_core) {
     /**
      * Get gradeable buckets array
      *
-     * @return array
+     * @return array<object>
```

Comment:
I may not understand, but do `@return` and `@var` types have different syntax ( array<string> vs string[] )?

Reply:
Both work. I made it consistent in [0dd3d65](https://github.com/Submitty/Submitty/pull/10871/commits/0dd3d65e0dc1506beb9955d6df8de354a7ee82af)